funfact/loss.py

```diff
@@ -12,14 +12,15 @@ def _loss(self, model, target):
         pass
 
     def __call__(self, model, target, reduction='mean', sum_vec=True,
-                 **kwargs):
-        if target.ndim == model.ndim - 1:
-            if model.shape[:-1] != target.shape:
-                raise ValueError(f'Target shape {target.shape} and '
-                                 f'model shape {model.shape[:-1]} mismatch.')
-            data_axis = tuple(i for i in range(target.ndim))
-            target = target[..., None]
-        elif target.ndim == model.ndim:
+                 append=True, **kwargs):
```

Comment:
I find the meaning of `append` a bit cryptic here. This is different from the situation of the `vectorize` method, whose name already provides the context. Here, the context is 'loss', and hence it is not very intuitive as to what 'append' would mean. What about `vectorizes_along_last` or something along the aline?

Reply:
Changed here to `vectorized_along_last`.